scene/3d/physics/joints/generic_6dof_joint_3d.cpp

```diff
@@ -196,6 +198,29 @@ void Generic6DOFJoint3D::_bind_methods() {
 	BIND_ENUM_CONSTANT(FLAG_MAX);
 }
 
+PackedStringArray Generic6DOFJoint3D::get_configuration_warnings() const {
+	PackedStringArray warnings = Joint3D::get_configuration_warnings();
+
+	const String physics_engine = GLOBAL_GET("physics/3d/physics_engine");
```

Comment:
I actually have a PR in store that does something similar in preparation for Jolt, and the fact that. The fact that we have no way of fetching the current physics engine's name, if not by checking the editor setting is a bit weird.